streaming_data_types/area_detector_ADAr.py

```diff
@@ -0,0 +1,203 @@
+from typing import Union, NamedTuple, List
+import flatbuffers
+from streaming_data_types.fbschemas.ADAr_ADArray_schema import ADArray
+from streaming_data_types.fbschemas.ADAr_ADArray_schema.DType import DType
+import streaming_data_types.fbschemas.ADAr_ADArray_schema.Attribute as ADArAttribute
+from streaming_data_types.utils import check_schema_identifier
+import numpy as np
+from datetime import datetime
+from struct import pack
+
+FILE_IDENTIFIER = b"ADAr"
+
+
+class Attribute:
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        source: str,
+        data: Union[np.ndarray, str, int, float],
+    ):
+        self.name = name
+        self.description = description
+        self.source = source
+        self.data = data
+
+    def __eq__(self, other):
+        data_is_equal = type(self.data) == type(other.data)  # noqa: E721
```

Comment:
isinstance will make the warning go away (and look nicer)

Reply:
probably

Reply:
`isinstance()` does not do what I want to do here.

Reply:
Fair enough